validator/testcases/javascript/instanceactions.py

```diff
@@ -166,6 +171,11 @@ def setAttribute(args, traverser, node, wrapper):
             description="To prevent vulnerabilities, event handlers (like "
                         "'onclick' and 'onhover') should always be defined "
                         "using addEventListener.",
+            signing_help="Please use `addEventListener` any place you might "
+                         "otherwise create event listener attributes. Event "
+                         "listener attributes will not be accepted in add-ons "
+                         "submitted for automated signing in any instance "
+                         "where they may be reasonably avoided.",
```

Comment:
Does that sentence imply that event listener attributes used in places where they may not be reasonably avoided (examples?) could get automatically signed?